bucket/openssl-light.json

```diff
@@ -15,9 +16,15 @@
             "hash": "69ae9e27c98a2ac87c210a22e1ab556ad65ab8b3a859b7f9cf1b109bd0530eac"
         }
     },
-    "innosetup": true,
+    "depends": "innounp",
+    "installer": {
+        "script": [
+            "Expand-InnoArchive -Path \"$dir\\$fname\"",
+            "Expand-InnoArchive -Path \"$dir\\$fname\" -DestinationPath \"$dir\\bin\" -ExtractDir '{cf}' -Removal",
```

Comment:
Why is this needed next to normal extraction?

Reply:
The default config file is in `{cf}` and will extract to `Common Files` during installation. So we should extract it and put them in the same folder as normal (not light) edition, i.e. `bin\cnf` (`{cf}\SSL`).

Normal edition also has `{cf}`, but it has `cnf` dir in `bin`, so there's no need to extract them. (See https://github.com/ScoopInstaller/Main/pull/251)